app/views/pages/cookies.md

```diff
@@ -1,126 +1,140 @@
+| `__zlcmid`                                    | Zendesk cookie that identifies your webchat session                                                                                                                                                                                                      | 1 year                                          |
+| `__zlcprivacy`                                | Zendesk cookie that stores your webchat cookie preferences                                                                                                                                                                                               | 1 year                                          |
+| `git-cookie-preferences-v1`                   | Website cookie that stores your cookie preferences                                                                                                                                                                                                       | 90 days                                         |
+| `_vis_opt_exp_{CAMPAIGNID}_goal_{GOALID}`     | This VWO cookie is created when a campaign goal is triggered. This cookie, when created, will always have a value of 1                                                                                                                                   | 100 days                                        |
+| `_vis_opt_test_cookie`                        | This VWO cookie is to read if cookies can be placed                                                                                                                                                                                                      | Browser closes                                  |
+| `_vis_opt_exp_{CAMPAIGNID}_combi`             | This VWO cookie is created when you are chosen for a particular variation for a test and ensures you see the same variation when you revisit a page in the future                                                                                        | 100 days                                        |
+| `_vis_opt_exp_{CAMPAIGNID}_exclude`           | This VWO cookie is created when you are excluded from a test campaign due to the traffic percentage specifications                                                                                                                                       | 100 days                                        |
+| `_vis_opt_exp_{CAMPAIGNID}_split`             | This cookie is created when one of the variations is chosen for you, but you are not yet redirected to the variation page. It exists so that VWO can verify that the variation page corresponds to the chosen variation after landing on the page        | 100 days                                        |
```

Comment:
```suggestion
| `_vis_opt_exp_{CAMPAIGNID}_split`             | VWO Cookie created when one of the variations is chosen for you, but you are not yet redirected to the variation page. It exists so that VWO can verify that the variation page corresponds to the chosen variation after landing on the page        | 100 days                                        |
```